app/services/workflow_service/property_assessment.rb

```diff
@@ -2,60 +2,70 @@ module WorkflowService
   class PropertyAssessment < BaseWorkflowService
```

Comment:
Most of the methods in this service pass in a property, and a single instance of the service does all the calculations for all the properties. This strongly suggests that the structure of this object can be improved.

Firstly, I would refactor the class so that a class method `call` receives `capital_summary`. Then have that method iterate through all the properties and initiate an instance of the service to handle just that property. So:

```ruby
def self.call(capital_summary)
  capital_summary.properties.sum { |property| new(property, capital_summary).assessed_equity }
end
```

Second, each property should knows whether it is the main home. So you don't have to split out separate methods do handle the processing of each property type, but rather just check the `main_home` attribute where it needs to be checked.

In this way this class can be simplified a lot.